doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change. They are not yet ready for user testing.
+> * 1.1 - Active development. Experimental features at this stage are nearing
+>   potential stability, with only minor changes expected.
+> * 1.2 - Release candidate. Experimental features at this stage are hopefully
```

Comment:
I'm not sure I understand the difference between this and stable. Will a feature belong to `Release candidate` if the developers of the feature think that it's stable but there's no way to confirm because of low adoption in userland / absence of WPT tests?

Reply:
The whole point of a feature marked as experimental is to be able to do breaking changes in a semver minor. As this is written, there is no difference from stable.

I would remove this.

Reply:
See above. We already have features we consider “baking,” that are essentially stable in all but name. We should document them as such, to encourage adoption of them by the slightly bolder of our user base. It sucks to get a bug report right after marking something as stable.